docs/YOSEMITE.md

```diff
@@ -38,7 +38,8 @@ At the moment, we provide the following subclasses of `Store`:
 * `AccountStore`. Registers `AccountAction` with the `Dispatcher`.  It implements the business logic necessary to manage an account (load an account, load a site, synchronise account information…)
 * `AppSettingsStore`. Registers and responds to actions declared in `AppSettingsAction`. It implements the logic to save and retrieve application settings.
 * `CommentStore`.  Registers and responds to actions declared in `CommentAction` It implements the business logic pertaining comments.
-* `CardPresentPaymentsStore`. Registers and responds to actions declared in `CardPresentPaymentsAction`. It imple ents the logic to collect payments with external card readers. 
+* `CardPresentPaymentsStore`. Registers and responds to actions declared in `CardPresentPaymentsAction`. It implements the logic to collect payments with external card readers.
+* `CustomerStore`. Implements support for operations related to Customers, like retrieving customer details.
```

Comment:
Nice one, good attention to detail! I've probably missed this for other stores in the past 😬